bridge/src/main/java/com/livefront/bridge/wrapper/BitmapWrapper.java

```diff
@@ -26,18 +25,25 @@ public Bitmap getBitmap() {
 
     //region Parcelable
     protected BitmapWrapper(Parcel in) {
+        int width = in.readInt();
+        int height = in.readInt();
+        Bitmap.Config config = Bitmap.Config.values()[in.readInt()];
         byte[] bytes = in.createByteArray();
-        mBitmap = BitmapFactory.decodeByteArray(
-                bytes,
-                0,
-                bytes.length);
+
+        mBitmap = Bitmap.createBitmap(width, height, config);
+        mBitmap.copyPixelsFromBuffer(ByteBuffer.wrap(bytes));
     }
 
     @Override
     public void writeToParcel(Parcel dest, int flags) {
-        ByteArrayOutputStream stream = new ByteArrayOutputStream();
-        mBitmap.compress(Bitmap.CompressFormat.PNG, 100, stream);
-        dest.writeByteArray(stream.toByteArray());
+        int size = mBitmap.getRowBytes() * mBitmap.getHeight();
```

Comment:
`getRowBytes` has the following warning:

```
* <p>As of {@link android.os.Build.VERSION_CODES#KITKAT}, this method
* should not be used to calculate the memory usage of the bitmap. Instead,
* see {@link #getAllocationByteCount()}.
```
Are you able to use `getAllocationByteCount` instead?

Reply:
Ah good call. I can double check that, yeah.

Reply:
Thanks for pointing this out. So I added a version check and now use either `getAllocationByteCount` or `getByteCount` depending on the version.

Reply:
Cool.  What's the minimum SDK for the library at this point?

Reply:
14. So pretty low. And before that I believe it was 1, but I needed to add something that required API 14.